docs/getting_started/troubleshooting.rst

```diff
@@ -131,7 +131,7 @@ partitions. Note, that to avoid excessive performance degradation, ``dtype`` val
 be set fine-grained as it possible (specify ``dtype`` parameter only for columns with
 heterogeneous data).
 
-Setting of ``dtype`` parameter works well for most of the cases, but, unfortunetely, it is
+Setting of ``dtype`` parameter works well for most of the cases, but, unfortunately, it is
```

Comment:
This seems a bit odd grammar-wise. Would do something like "[...] for most of the cases; unfortunately, it is [...]"

Reply:
```suggestion
Specifying the ``dtype`` parameter will work well in most cases. Unfortunately, it is
```